src/main/kotlin/org/aarbizu/baseballDatabankFrontend/db/QueryEngine.kt

```diff
@@ -43,12 +39,11 @@ class QueryEngine(private val dbProvider: DBProvider) {
         ): List<TableRecord> {
             val timer = Stopwatch.createStarted()
             val rs = statement(dbProvider, queryTemplate, binds).executeQuery()
-            timer.reset()
             logger.info("exec query: $timer")
+            timer.reset()
             timer.start()
             val records: List<TableRecord> = extractor.invoke(rs)
             logger.info("extract results: $timer")
-            timer.stop()
```

Comment:
any reason to stop it? `timer` is about to go out scope anyway

Reply:
excessive pedantry :P